web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java

```diff
@@ -86,6 +86,26 @@
 	
 	private static final org.slf4j.Logger log = LoggerFactory.getLogger(InitializationFilter.class);
 	
+	/**
+	 * Supported Database
```

Comment:
What the value added by this comment?

Reply:
My motive behind adding this string was to tell the one who see's the code to show the database that openmrs supports currently. I think I'll remove this string over H2 and sqlserver. What do you think should I add this string or remove it from everywhere ?

Reply:
If it the the same comment used for four different things, then i do not see the value added.